### DISCUSSION TITLE
Freezes on Launch Restoring Previous Session Layout

### DISCUSSION THREAD

**Discussion by zamostny** [Eugeny/tabby#5173] [Q&A]
Tabby now freezes when trying to laucnh and load the previous layout/session. Is there a way to clear this or should I re-install?

**Comment by Eugeny** [comment]
Frozen restored tabs are fixed in v168

**Reply by zamostny** [reply]
> Frozen restored tabs are fixed in v168

Just installed that new release and it still is freezing at launch. 
![tabby crash](https://user-images.githubusercontent.com/55537962/146223589-d5454815-3229-468c-a1c7-8c5fd4f4d7b9.png)

**Reply by zamostny** [reply]
@Eugeny Is there a log file I could provide you? Anything I can do to help, let me know.

**Reply by Eugeny** [reply]
Last session is stored in `%APPDATA%\Tabby\Local storage` - you can just nuke the folder. If this doesn't help and the session freezes again, it'd be helpful to know whether the whole app is frozen or just the terminal panes, and whether there's anything logged in Devtools (Settings -> Application -> Devtools)

**Reply by zamostny** [reply]
> Last session is stored in `%APPDATA%\Tabby\Local storage` - you can just nuke the folder. If this doesn't help and the session freezes again, it'd be helpful to know whether the whole app is frozen or just the terminal panes, and whether there's anything logged in Devtools (Settings -> Application -> Devtools)

Well nuking the contents of that folder worked for a minute... but the entire app froze again once I tried to open a new terminal. So I nuked the folder contents again, relaunched and open the DevTools. See attached. This is immediately after launching the app, no terminals open. 
![tabby crash 2](https://user-images.githubusercontent.com/55537962/146241267-78d499d3-c398-4748-9cea-80b256b25f42.png)

**Reply by Eugeny** [reply]
Could you please try the build linked at the bottom at this page? https://github.com/Eugeny/tabby/actions/runs/1569560327

**Reply by zamostny** [reply]
@Eugeny Same issue. No matter what kind of terminal I select, the app becomes unresponsive. Down to the window buttons, nothing is clickable.

**Reply by zamostny** [reply]
@Eugeny I got the sad face

**Reply by Eugeny** [reply]
Do you have font ligatures enabled in Settings -> Appearance? Another user had a similar issue with it on.

**Reply by zamostny** [reply]
> Do you have font ligatures enabled in Settings -> Appearance? Another user had a similar issue with it on.

This was the issue! Works great now!

**Reply by Eugeny** [reply]
Thanks for confirming! You can subscribe to track the progress on that issue in #5188
